### DISCUSSION TITLE
flowise on private cloud

### DISCUSSION THREAD

**Discussion by amitpanda123** [FlowiseAI/Flowise#3505] [General]
I am trying to deploy flowise on my company private cloud and even after setting up the GLOBAL_AGENT_HTTP_PROXY it seems it doesn't connect to internet. I am using the SerpAPI to search something from internet using the LLM but it doesn't connect to the serp website while running inside the pod. However I am able to access internet from the pod by setting up HTTP_PROXY but it seems the flowise process doesn't have access to internet. How can i fix it when the pod is able to connect to outside internet but the flowise process can't.

**Comment by dni-coder** [comment]
Hello amitpanda123, are you able to solve this problem?
Im still facing this issue and have no workaround right?

Regards
David
